frontend/src/metabase-types/api/mocks/dataset.ts

```diff
@@ -21,24 +25,29 @@ export type MockDatasetOpts = Partial<Omit<Dataset, "data">> & {
 export const createMockDataset = ({
   data = {},
   ...opts
-}: MockDatasetOpts = {}) => ({
-  data: {
-    rows: [],
-    cols: [
-      createMockColumn({
-        display_name: "NAME",
-        source: "native",
-        name: "NAME",
-      }),
-    ],
-    rows_truncated: 0,
-    ...data,
-  },
-  database_id: 1,
-  row_count: 0,
-  running_time: 1000,
-  ...opts,
-});
+}: MockDatasetOpts = {}) => {
+  const columns: DatasetColumn[] = data.cols ?? [
+    createMockColumn({
```

Comment:
I wonder, are the params are necessary here for the column?

Reply:
Yeah looks weird, I'd rather remove them and rely on the defaults if possible. But that's not an issue with the PR